README.md

```diff
@@ -24,8 +24,8 @@ module "waf" {
 
   alb_arn                             = "${module.alb_web_containers.alb_arn}"
   associate_alb                       = true
-  ip_rate_limit                       = 2000
   ip_sets                             = [aws_wafregional_ipset.global.id]
+  rate_based_rules                    = [aws_wafregional_rate_based_rule.ipratelimit.id]
   regex_host_allow_pattern_strings    = var.waf_regex_host_allow_pattern_strings
```

Comment:
Remove deprecated `regex_host_allow_pattern_strings` and `regex_path_disallow_pattern_strings`